include/pmacc/mappings/threads/ThreadCollective.hpp

```diff
@@ -30,6 +30,15 @@
 
 namespace pmacc
 {
+    struct DefaultApplyOffset
```

Comment:
Changes in this file are not required if pmacc reduction method will be used.
offline discussed that this was used as a workaround to write its own reduction method.